src/globus_sdk/services/auth/response/oauth.py

```diff
@@ -250,12 +250,49 @@ def __str__(self) -> str:
         )
 
 
+class AuthorizationCodeTokenResponse(OAuthTokenResponse):
```

Comment:
\<NamingWar>

I think it's a step too far to remove the OAuth prefix from these names. The existing 2 response classes + the many AuthLoginClient methods have established the precedent of prefixing this category of things with OAuth (or oauth2 in the case of methods) and even if I like the shorter names you set up here a bit better; I'll advocate that we should be consistent for relationship-hinting's sake.

My formal change submission would be:
 1. Add the prefix "OAuth" to label-wise consistent
 2. Replace the suffix "TokenResponse" with "Response".

These two work with  the existing custom grant-type response class (DependentToken) to set a clear naming pattern of `f"OAuth{GrantType}Response[OAuthTokenResponse]"`

---

If you buy that, we'd end up with:
```python
class OAuthAuthorizationCodeResponse[OAuthTokenResponse]: ...
class OAuthClientCredentialsResponse[OAuthTokenResponse]: ...
class OAuthDependentTokenResponse[OAuthTokenResponse]: ...
class OAuthRefreshTokenResponse[OAuthTokenResponse]: ...
```

Reply:
> If you buy that

I can buy my way out of naming wars? Hush and take my money! :joy: 

---

I'm perfectly okay with your suggestion and will apply it. There's a mild discrepancy with the base class, but I don't mind it much.

Even if we had space for a clean-room design for how to name the responses for different variants on this API, I think we might struggle to determine what naming is best.

Reply:
Yeah, I think if it were a clean namespace, I'd probably suggest that the parent class (and suffix?) be GrantResponse or OAuthGrantResponse but that's not really feasible right now.